src/providers/deepgram.py

```
@@ -36,6 +36,62 @@ def _log_provider_task_exception(task: asyncio.Task) -> None:
+
+    Pre-v6.5.0 the listen model was hardcoded to ``nova-3`` regardless of
+    config. v6.5.0 makes the model honor config and additionally builds a
+    Flux-aware payload when the configured model starts with ``flux``,
+    matching Deepgram's published Voice Agent configuration:
+    https://developers.deepgram.com/docs/configure-voice-agent
+
+    Behavior:
+      - Nova-* models (default): emit ``{"type": "deepgram", "model": <model>}``.
+      - Flux models (``flux-general-en``, ``flux-general-multi``): additionally
+        emit ``version: "v2"`` (required) plus the Flux-specific tuning fields
+        ``eot_threshold`` (default 0.7, valid range 0.5–0.9), optional
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)